terraform/modules/ecr/provisioner.initial-image.tf

```diff
@@ -0,0 +1,11 @@
+resource "terraform_data" "initial_image_provisioner" {
```

Comment:
Puts the dummy alpine image in the ECR so there is something for the data resource to pull from initially. The next step is always populating the ECRs with the real artifact so this only exists to stand up that first image for the workaround